Fluid/Values/FluidValue.cs

```diff
@@ -156,8 +158,14 @@ public static FluidValue Create(object value, TemplateOptions options)
                         case DateTimeOffset dateTimeOffset:
                             return new DateTimeValue(dateTimeOffset);
 
+                        case IFormattable formattable:
+                            return new StringValue(formattable.ToString(null, options.CultureInfo.NumberFormat));
```

Comment:
should the parameter be `options.CultureInfo` instead of `.NumberFormat`? `FormattableString` is IFormattable and there number format sounds odd

Reply:
Thanks, somehow I missed it could accept CultureInfo directly, it confused me.